Marker/Classes/Marker.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  Marker.swift
+//  Marker
+//
+//  Created by Htin Linn on 5/15/17.
+//  Copyright © 2017 Prolific Interactive. All rights reserved.
+//
+
+import Foundation
+
+/// Returns formatted Markdown text as an attributed string.
+///
+/// - Parameters:
+///   - markdownText: String with Markdown tags.
+///   - textStyle: Text style object containing style information.
+/// - Returns: Formatted Markdown text.
+public func attributedMarkdownString(from markdownText: String,
```

Comment:
I did consider this approach but it comes with its own downsides. Giving a type the same name as the module name make types inaccessible using the module prefix. 

For example, if I define a `Result` type as part of the `Marker` module and you want to reference this type from a file where you also import `Alamofire`, you won't be able to. `Result` by itself would be ambiguous because both modules have the type `Result` and `Marker.Result` wouldn't work either because the compiler will be looking for the nested `Result` type inside the `Marker` type. 

P.S. I looked up and `request` function from `Alamofire` is a free function just like these two.